db/migrate/20170313161311_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+class CreateArticles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.string :description
+      t.string :locale
```

Comment:
It is likely to be implemented, but it is not too urgent. It is good that all the posts are created now with either `es` or `en`.